docs/dev/table/connect.md

```diff
@@ -127,7 +128,15 @@ schema: ...
 
 <div data-lang="DDL" markdown="1">
 {% highlight sql %}
-tableEnvironment.sqlUpdate("CREATE TABLE MyTable (...) WITH (...)")
+tableEnvironment.sqlUpdate("CREATE TABLE MyTable (...) 
+WITH (
+'connector.key0'='...'
+...
+'connector.keyn'='...'
+'update-mode'='append'
+'format.key0'='...'
+...
+'format.keyn='...')")
```

Comment:
I don't think we should change this. 
1. It's hard to guess what is `key0` , `keyn`. 
2. not every connector requirs `update-mode` and `format.*`.

Reply:
@wuchong thanks for you review.
I agree (1) that key0, keyn is a little hard to understand, and I knew (2) yet. 
But in this doc, we firstly introduce a full connector definition in different way(i.e. java/scala api, python api, YAML, DDL),  all of them cover each definition part (connector, format, and schema) except DDL, then we introduced these part one by one.
So, I think we should change the DDL part make it more cleat and follow same definition style.

post other definition contains these part:

java/scala api:
```
tableEnvironment
  .connect(...)
  .withFormat(...)
  .withSchema(...)
  .inAppendMode()
  .createTemporaryTable("MyTable")
```
YAML:
```
name: MyTable
type: source
update-mode: append
connector: ...
format: ...
schema: ...
```
DDL which misses these parts  
```
tableEnvironment.sqlUpdate("CREATE TABLE MyTable (...) WITH (...)")
```